### ISSUE TITLE
conda.exceptions.InvalidSpecError: Invalid spec: =20190808

### ISSUE THREAD

**Issue by atabbal** [conda/conda-build#3915]
I have a rather complex build workflow that I’m still investigation. However, here’s what I know:

It uses conda, conda-build, conda-build-all, and miniconda to create python environments from meta.yaml files. 

These are large (>100 packages) meta files that result in a >400 library environment.

Admittedly, it’s a miracle that conda ever resolves a combination that works together. 

Starting last week (12-March-2020) the build started failing with a [not-very-helpful] message:
`conda.exceptions.InvalidSpecError: Invalid spec: =20190808`

It is not using he latest `conda` version and updating isn’t straightforward (multiple build scripts, systems, etc.) I know the whole thing need to be  re-engineered however, that’s not my first choice right  now. 

Versions used:
```
conda-4.3.34-py36_0 ...
conda-build-2.1.18-py36_0 ...
conda-build-all-1.1.1-py36_1 ...
conda-env-2.6.0-0 ...
conda-verify-3.1.1-py36_1000 ...
```

Any suggestion as to how to troubleshoot and fix this error?

**Comment by atabbal** [comment]
@msarahan  -- what information would help the community debug or provide suggestions?

**Comment by Kethaias** [comment]
We are experiencing the same issue with our environment. There seem to be a number of posts with this exact same error message, but none have been resolved yet.

**Comment by jjhelmus** [comment]
@atabbal conda 4.3.34 and conda-build 2.1 are both very old, can you try updating to a newer versions and see if the problem persists?

**Comment by bklaas** [comment]
Also seeing this error frequently. We've got a multi-user shared conda (4.3.22) installation in production. 

Though I'm resigned it may be our only option, upgrading is not a simple path for us. I think this should be considered a regression against previously working versions of conda.

**Comment by Kethaias** [comment]
While certainly not ideal, we were able to get a workaround working.

We installed a newer version of conda in a different location. It seems that if the environment is activated with the newer version of conda (even if the older one was the one which created it) installs are fully functional again.

@bklaas - Not sure if this would be any easier for you than upgrading the base install to a newer version; I know it was for us
